apps/console/src/features/identity-providers/data/identity-provider-templates/templates/saml-identity-provider/saml-idp-wizard-file-based-help.tsx

```diff
@@ -21,6 +21,7 @@ import { Code, Heading } from "@wso2is/react-components";
 import React, { FunctionComponent, ReactElement } from "react";
 import { AppState, ConfigReducerStateInterface } from "../../../../../core";
 import { useSelector } from "react-redux";
+import {Divider} from "semantic-ui-react";
```

Comment:
Lint issue here. Check for import ordering as well.
```suggestion
import { Divider } from "semantic-ui-react";
```